src/queue.hpp

```diff
@@ -172,11 +172,21 @@ struct cvk_command_queue : public _cl_command_queue,
         return m_command_pool.free_command_buffer(cmdbuf);
     }
 
+    size_t get_printf_buffer_size() {
+        auto& properties = m_context->properties();
+        for (unsigned i = 0; i < properties.size(); i += 2) {
```

Comment:
This is kind of a duplicate of `cvk_command_queue::get_printf_callback`. Maybe merge that in a function in `cvk_context`, something like `get_property` or `get_property_index`.